bwe/RateController.cpp

```diff
@@ -167,9 +167,7 @@ void RateController::onReportBlockReceived(uint32_t ssrc,
     {
         if (!_lastProbe.empty() && item.reportNtp == _lastProbe.ntp && item.type == PacketMetaData::SR)
         {
-            // probes before last probe may have been updated.
-            _lastProbe.packetsReceived = 0;
-            _lastProbe.ntp = 0;
+            _mayHaveBetterProbeThanLast = true;
```

Comment:
Do not throw away the _lastProbe. Just mark as we may have better probes than the last to be consider

Reply:
Are we right about this lastProbe, regardless of this change?
If we have SR1 .... SR2....RB2a...RB1...RB2b, we may have selected SR2 as last probe but since we confirmed packets  between RB2a and RB2b, we may have improved both SR1 and SR2.

Reply:
Yes, it why mark this as "_mayHaveBetterProbeThanLast" and this will required to analyse all backlog (not only until _lastProbe) . Then we are giving the opportunity to SR2 to be analysed again. And even if it SR1 os chosen again, it only be considered the same if it has confirmed the same number of the last time.

Reply:
I was just thinking it will cause a deeper search if we get a report referencing to the previously selected probe, but if the report is affecting both the last and previous probe we do not examine both. At RB1 for example.